proto/spacemesh/v1/smesher_types.proto

```diff
@@ -57,16 +57,16 @@ message CoinbaseResponse {
     AccountId account_id = 1;
 }
 
-message EstimatedRewardsRequest {}
+message EstimatedRewardsRequest {
+    uint32 layer_id = 1;
+}
 
 // Estimated rewards for the next epoch. Note that this is a global value that will be the
 // same for all smeshers, and is based on the default minimum commitment size and fixed
 // epoch length, both of which are network params.
 message EstimatedRewardsResponse {
     // The amount of the total estimated reward in the next upcoming epoch
     Amount amount = 1;
-    // The number of Post data commitment units that this estimated reward corresponds to (part of global config)
-    uint32 num_units = 2;
```

Comment:
are you sure that it needs to be removed?

Reply:
`NumUnits` looks like is how much space was allocated, not looks like it really need for this endpoint.
We can add it later, if it will be need for some reasons

Reply:
Is this comment still accurate?

> Note that this is a global value that will be the
> // same for all smeshers, and is based on the default minimum commitment size and fixed
> // epoch length, both of which are network params.